provider-ci/src/goreleaser.ts

```diff
@@ -113,6 +113,10 @@ export class PulumiGoreleaserPreConfig implements GoreleaserConfig {
       ];
     }
 
+	if (opts.provider === "newrelic") {
+		ldflags.push(`-X github.com/pulumi/pulumi-${opts.provider}/provider/v${opts["major-version"]}/pkg/useragentservicename.UserAgentServiceName="pulumi"`)
```

Comment:
I think we would want to update the newrelic module, not pulumi-newrelic: 


```suggestion
		ldflags.push(`-X github.com/newrelic/terraform-provider-newrelic/v2/main.UserAgentServiceName=pulumi`)
```

Could we write a unit test to verify that works, since `main.UserAgentServiceName` is public?